REFERENCES.md

```diff
@@ -0,0 +1,87 @@
+# Reference Material
+
+The reference material entries listed in this document are contributions covering GitOps principles and related topics. They do not constitute endorsements or affiliations with any specific tools, solutions, or companies mentioned in these documents. The inclusion of references is purely for academic or informational purposes to support understanding and research in the field of GitOps. If you want to include any related material please submit a PR.
```

Comment:
PS I know we discussed this again at the last OpenGitOps meeting, but seriously bravo 👏 for organizing this page. This is EXACTLY what we need.

Reply:
Thanks Scott, I have added some entries already of books I went thru and research papers I read about gitops and left the articles, videos, documentation links to be added by others and eventually getting PRs to add more

Reply:
I'm thinking more about where the REFERENCES.md file should live. Perhaps that should be in the [github.com/open-gitops/project](http://github.com/open-gitops/project) repo rather than the /documents repo. And be renamed to something like "GitOps community resources page," as a solution to this discussion https://github.com/open-gitops/project/discussions/99? That way that one can continue to grow totally independently of the versioned documents

Reply:
noting this is done ✅ in https://github.com/open-gitops/project/pull/194